_templates/plugin_layouts/sphinx_bootstrap/globaltoc.html

```diff
@@ -3,7 +3,7 @@
 <li class="dropdown globaltoc-container">
   <a href="{{ pathto(master_doc) }}"
      class="dropdown-toggle"
-     data-toggle="dropdown">{{ theme_navbar_site_name }} <b class="caret"></b></a>
```

Comment:
These changes are fine to include in fopp BUT they should also be added to `RunestoneComponents/runestone/common/project_template/_templates/....` so that new projects will get the benefit of your work and existing projects can also do a runestone update to install new versions of the templates with these improvements.